.github/workflows/lint-test.yml

```
@@ -0,0 +1,106 @@
+      continue-on-error: true
+
+    - name: Generate test summary
+      if: always()
+      uses: actions/github-script@v7
+      with:
+        script: |
+          const fs = require('fs');
+          const core = require('@actions/core'); // core 임포트 추가
+          try {
+            // 커버리지 결과 파일이 있는지 확인
+            const coverageSummary = fs.existsSync('./coverage/coverage-summary.json') 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)